app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = Post.where(author_id: @user.id).order(created_at: :desc).limit(3)
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly use `most_recent_3_posts` method you have created in user model. it should look something like this: 

```ruby
@posts = @user.most_recent_3_posts